lib/experimental/Widgets/Content/TasksList/TaskItem/index.tsx

```diff
@@ -0,0 +1,63 @@
+import { Badge } from "@/components/Information/Badge"
+import { Counter } from "@/experimental/Information/Counter"
+import { DottedCircle, InProgressTask } from "@/icons"
+import { useCallback } from "react"
+
+export type TaskStatus = "in-progress" | "due" | "no-due"
+export interface Task {
+  id: number | string
+  text: string
+  badge?: { text: string; isPastDue?: boolean }
+  counter?: number
+}
+
+export type TaskItemProps = {
+  task: Task
+  status: TaskStatus
+  onClick?: (task: Task) => void
+  hideIcon?: boolean
+}
+
+export function TaskItem({
+  task,
+  status,
+  onClick,
+  hideIcon = false,
```

Comment:
yeah.. we don't want the icon in jobs to be done, because jobs never ends.. you can clean to 0, but tomorrow you will have X new invoices to review, etc

Reply:
Figma: https://www.figma.com/design/sjXPhNmfJI3OnVpWmfn6Lc/Accounts-Payable?node-id=3538-63797&node-type=frame&t=b8j7yGGhmAThnI8l-0